packages/labs/src/components/select/multiSelect.tsx

```diff
@@ -33,14 +33,19 @@ export interface IMultiSelectProps<T> extends IListItemsProps<T> {
     selectedItems?: T[];
 
     /**
+     * React child to render when query is empty.
+     */
+    initialRenderer?: JSX.Element;
```

Comment:
again, not the correct name. why is this in this PR?

looks like _someone_ didn't branch from `master` 😉